AK/HashMap.h

```diff
@@ -54,13 +54,16 @@ class HashMap {
     size_t capacity() const { return m_table.capacity(); }
     void clear() { m_table.clear(); }
 
-    void set(const K& key, const V& value) { m_table.set({ key, value }); }
-    void set(const K& key, V&& value) { m_table.set({ key, move(value) }); }
-    void remove(const K& key)
+    bool set(const K& key, const V& value) { return m_table.set({ key, value }); }
```

Comment:
I would never for the life of mine guess what it means if `HashMap::set()` returns `false` — it failed to set?

If we want to know whether the key was already present, we could make a special enum, perhaps like this:

```cpp
enum SetResult {
    InsertedNewEntry,
    ReplacedExistingEntry
};
```

Reply:
Good idea, I like it